testplan/testing/multitest/driver/base.py

```diff
@@ -134,7 +129,7 @@ def get_options(cls):
             ConfigOption("stdout_regexps", default=None): Or(None, list),
             ConfigOption("stderr_regexps", default=None): Or(None, list),
             ConfigOption("file_logger", default=None): Or(None, str),
-            ConfigOption("async_start", default=False): bool,
+            ConfigOption("async_start", default=UNSET): Or(UNSET_T, bool),
```

Comment:
I think there's no need to touch the async_start flag. If dependencies are specified, just disregard whatever is in async_start?

Reply:
What was the conclusion on this?